nf_core/lint.py

```diff
@@ -237,18 +236,14 @@ def check_files_exist(self):
             os.path.join('docs','README.md'),
             os.path.join('docs','output.md'),
             os.path.join('docs','usage.md'),
-            ['.travis.yml', os.path.join('.github', 'workflows', 'branch.yml')],
-            ['.travis.yml', os.path.join('.github', 'workflows','ci.yml')],
-            ['.travis.yml', os.path.join('.github', 'workflows', 'linting.yml')]
-
+            os.path.join('.github', 'workflows', 'branch.yml'),
+            os.path.join('.github', 'workflows','ci.yml'),
+            os.path.join('.github', 'workflows', 'linting.yml')
         ]
         files_warn = [
             'main.nf',
             'environment.yml',
-            os.path.join('conf','base.config'),
-            os.path.join('.github', 'workflows', 'branch.yml'),
-            os.path.join('.github', 'workflows','ci.yml'),
-            os.path.join('.github', 'workflows', 'linting.yml')
+            os.path.join('conf','base.config')
         ]
         files_fail_ifexists = [
             'Singularity'
```

Comment:
We should move Travis here, so that tests fail if the config file is found.

Reply:
Yeah, I did think that too but then I went for leniency 👍

Reply:
Also, trying to factor in edge cases like @MaxUlysse where he needs to use other CI platforms.

Reply:
Tempted to leave this in to avoid failures if `.travis.yml` is used for other purposes.